browser-sign-in/App.js

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (c) 2018, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+ 
+import React from 'react';
+
+import { Button, StyleSheet, Text, View } from 'react-native';
+import { 
+  createConfig, 
+  signIn, 
+  signOut, 
+  getAccessToken, 
+  isAuthenticated,
+  getUser,
+  getUserFromIdToken,
+  EventEmitter
+ } from '@okta/okta-react-native';
+ import configFile from './samples.config';
+
+export default class App extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      authenticated: false,
+      context: null
+    }
+    this.checkAuthentication = this.checkAuthentication.bind(this);
+    this.getMessages = this.getMessages.bind(this);
+  }
+  
+  async componentDidMount() {
+    let that = this;
+    EventEmitter.addListener('signInSuccess', function(e: Event) {
+      that.setState({ authenticated: true });
+      that.setContext('Logged in!');
+    });
+    EventEmitter.addListener('signOutSuccess', function(e: Event) {
+      that.setState({ authenticated: false });
+      that.setContext('Logged out!');
+    });
+    EventEmitter.addListener('onError', function(e: Event) {
+      console.log(e);
```

Comment:
Can we show alert with error?

Reply:
outputs console.error(e) instead. Also shows error with setContext